lib/verifiablePresentation/SSI.types.ts

```diff
@@ -0,0 +1,72 @@
+import { PresentationSubmission } from '@sphereon/pe-models';
+
+export interface CredentialSubject {
+  [x: string]: unknown;
+}
+
+export interface Proof {
+  type: string;
+  created: string;
+  proofPurpose: string;
+  verificationMethod: string;
+  jws: string;
+  [x: string]: string;
+}
+
+export interface CredentialStatus {
+  id: string;
+  type: string;
+}
+
+export interface Credential {
+  '@context': string[];
+  id: string;
+  type: string[];
+  credentialSubject: CredentialSubject;
+  issuer: string;
+  issuanceDate?: string;
+  expirationDate?: string;
+  credentialStatus?: CredentialStatus;
+  [x: string]: unknown;
+}
+
+export interface VerifiableCredential extends Credential {
+  issuer: string;
+  issuanceDate: string;
+  proof: Proof;
+}
+
+export interface Presentation {
+  '@context': string[];
+  type: string;
+  verifiableCredential: VerifiableCredential[];
+  holder: string;
+}
+
+export interface VerifiablePresentation extends Presentation {
```

Comment:
you have another interface in this directory with the same name. if we are using this like this, we probably don't want this here. on another note, @hrehman-sphereon don't we want this types to be in the `pe-models`?

Reply:
@sksadjad I saw it after opening the pull request, TS/JS does not complain about duplication. What we have now are different structures in pe-js, didsiop library and pe-models. I need to check if it is possible to create a single structure to be used by all of them of if it is necessary to use a context based approach like domain driven design does.

Reply:
@ksadjad By the way, I've found something interesting about TS interfaces: https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces

Reply:
PE JS does not deal with proof/claim related objects.
These objects can be out of the pe-models.

My opinion: I do not mind it either way:
1) keeping it in PE-JS.
2) moving these to some other project ... may be call them verifable presentation models.
3) justifying to move to pe-models may be a bit of stretch.